ibis/backends/base/sqlglot/compiler.py

```diff
@@ -155,6 +156,7 @@ def __init__(self) -> None:
         self.f = FuncGen()
         self.v = VarGen()
 
+    # TODO(kszucs): actually the type mapper also contains the dialect
```

Comment:
There's nothing to do here AFAICT. The duplicated `dialect` property is there to reduce the amount typing.

Reply:
Removed the note.